### PR TITLE
fix: dist bar will render value multi times

### PR DESCRIPTION
**Pull request by Mikedu1988**
🐛 Bug Fix
related issue: https://github.com/apache-superset/superset-ui/issues/712

Now irrelevantt records will be removed, only the last applied breakpoints will be rendered.

### CODE REVIEW

**Comment by Mikedu1988** [comment]
@ktmud

File: package.json [hunk 1]
```diff
@@ -78,7 +78,8 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-test-renderer": "^16.13.1",
-    "rimraf": "^3.0.2"
+    "rimraf": "^3.0.2",
+    "uuid": "^8.3.0"
```
Comment (ktmud) [inline package.json:82]: why is this needed?

Comment (Mikedu1988) [reply package.json:82]: I think this is added during debug, it's not needed. I'll push one commit again.

**Comment by Mikedu1988** [comment]
juse one quick question, we forked and built our own superset docker image, how can we pick up this bug fix? should we just try a rebuild? or we need to bump the version for the plugin here and then we build our image based on the newer version. @ktmud

**Comment by villebro** [comment]
@Mikedu1988 this is now released as version `0.14.21` of `@superset-ui/legacy-preset-chart-nvd3` (will be bumped in main Superset repo shortly)
